Lab1_Operationalizing_Pytorch_with_Mlflow/README.md

```diff
@@ -1,31 +1,140 @@
-# Lab #1 : Operationalizing Pytorch Model with Mlflow
+# Operationalizing Pytorch Model with Mlflow
 
-## Scenario
+## Context
 
-PyTorch is one of the main machine learning library used for applications such as computer vision and natural language processing. 
-It is catching up with TensorFlow accelerating the path from research prototyping to production deployment.
-Recently, Pytorch community announched a number of technical contributions to enable end-to-end support for MLflow usage with PyTorch, 
-including support for: 
+MLOps is a new buzzword of Software Development. It goes beyond [DevOps](https://en.wikipedia.org/wiki/DevOps) 
+by adding a brand new perspective over data and models. In fact, Any reasonable Machine Learning
+solution must deal with assets like ML Models, datasets, features, etc. Therefore, a simple ML 
+application may turn into a complex workflow in order to ensure the best model runs in Production.
+Besides that, developing ML models demand a relevant time assessing and comparing parameters, algorithms
+and model performance. MLOps goal is to address these concerns by tying tools and ML frameworks in order to quickly 
+deploy models to Production (and keep them there).
 
-- autologging via PyTorch Lightning
-- TorchServe integration through a new deployment plug-in
+PyTorch is one of the main machine learning libraries used in applications such as computer vision and 
+natural language processing. It is catching up with TensorFlow accelerating the path from research 
+prototyping to production deployment. Recently, Pytorch community announced a number of technical 
+contributions to enable end-to-end support for MLflow usage with PyTorch. 
 
-and a sample end-to-end workflow targeting HuggingFace Transformers.
+MLflow is an open source platform to manage the ML lifecycle, including experimentation, reproducibility, 
+deployment, and a central model registry. One may interact with its components by command line interface
+or well-known APIs (Python, R, Java and REST), building a workflow capable of handling development and
+production activies. Actually, Its simple design allows small teams (or a lone cowboy data scientist as well) 
```

Comment:
Ha, cowboy might be a bit insulting out of context!!
```suggestion
production activies. Actually, Its simple design allows small teams (or a lone data scientist as well) 
```

Reply:
"Cowboy" removed. Still wanna be a cowboy something... maybe bc old western movies... haha